app/api/helpers/order.py

```diff
@@ -73,7 +74,14 @@ def create_pdf_tickets_for_holder(order):
             save_to_db(holder)
 
         # create order invoices pdf
-        order_ticket_info = OrderTicket.query.filter_by(order_id=order.id).one()
+        order_ticket_info_object = OrderTicket.query.filter_by(order_id=order.id)
```

Comment:
Why are there more than one and why only select first if more than one are found?

Reply:
There might be multiple tickets associated with an order. OrderTicket model contains `ticket_id`, `order_id` . But in the end, the order_invoice pdfs associated with the one who places the order.